package.json

```diff
@@ -19,23 +19,20 @@
   },
   "homepage": "https://github.com/mothras-cloud-solutions/study-stack#readme",
   "dependencies": {
+    "@faker-js/faker": "^8.4.1",
     "@types/express": "^4.17.21",
     "axios": "^1.6.8",
     "cors": "^2.8.5",
     "dotenv": "^16.4.5",
+    "fs": "^0.0.1-security",
```

Comment:
This package is not currently in use, see [https://www.npmjs.com/package/fs](https://www.npmjs.com/package/fs) - fs is automatically installed as part of node.js so I'm not clear why this would be needed here